app.rb

```diff
@@ -0,0 +1,54 @@
+require 'sinatra/base'
+require 'sinatra/reloader'
+require './lib/message'
+require './lib/user'
+require './lib/database_connection'
+require_relative 'database_connection_setup'
+
+class Chitter < Sinatra::Base
+  enable :sessions
+
+  configure :development do
+    register Sinatra::Reloader
+  end
+
+  get '/' do
+    @user = session[:user]
```

Comment:
storing the whole user object in session can create issues because session can't be very big. it might be better to just store user id in session then find the corresponding user when you need them